doc/Configuration.md

```diff
@@ -17,7 +17,7 @@ List of tenants allowed to use the system. Cronus is designed with multitenancy
 * Event Store - every tenant has a separate storage
 * Projection Store - every tenant has a separate storage
 
-Example: `"tenant1,tenant2,tenant3"`
+Example: `"["tenant1","tenant2","tenant3"]"`
```

Comment:
@tokmac we nee to include which symbols are allowed. Also specifying the regex here will be helpful